package.json

```diff
@@ -11,7 +11,8 @@
   "scripts": {
     "prepack": "npm run clean && npm run compile",
     "compile": "tsc --build tsconfig.json",
-    "clean": "rm -rf dist"
+    "clean": "rm -rf dist",
+    "test": "vitest run"
```

Comment:
Happy to swap this out for a different test runner, or remove it altogether if you prefer that!